src/components/Frame/components/Toast/Toast.tsx

```diff
@@ -69,11 +71,15 @@ export function Toast({
   const className = classNames(styles.Toast, error && styles.error);
 
   return (
-    <div className={className}>
-      <KeypressListener keyCode={Key.Escape} handler={onDismiss} />
-      {content}
-      {actionMarkup}
-      {dismissMarkup}
-    </div>
+    <ThemeProvider
+      theme={{mode: useTheme().mode === 'light' ? 'dark' : 'light'}}
```

Comment:
Going to use this a few times, so maybe a sharable `inverse(mode: Mode)` function? Nice to be able to read the word inverse.